### DISCUSSION TITLE
Hickups when importing old config from 2.3.0 into 3.0.0

### DISCUSSION THREAD

**Discussion by azido** [nhielost/obs-midi-mg#91] [General]
As you wished, here is the config i imported to 3.0.0 (works flawlessly on 2.3.0 on daily usage)
[obs-midi-mg-config.json](https://github.com/nhielost/obs-midi-mg/files/13628091/obs-midi-mg-config.json)

When i move sliders, this is what happens:
![Unbenanntes-Video-–-Mit-Clipchamp-erstellt-_1_](https://github.com/nhielost/obs-midi-mg/assets/151363/56c39ff1-6eac-4c2b-8433-bdcd94a1ecc1)

Also, cpu usage is now around 20% while doing nothing at all.

**Comment by nhielost** [comment]
This is very likely an issue dealing with so many responses to bindings that it overwhelms OBS a little bit, causing the stutters and the performance issues. (I was not able to get levels as high as 20%, but they were higher than usual.) With the new interface you are helping me develop, I am planning to change the response location from the binding to the message so that the number of responses decreases.

However, this is not to say that 82 bindings is a not a lot. Based on a quick scan of your configuration, there are things that you could do to decrease the number of bindings you have. For example, the Start Stream and Stop Stream bindings can be combined into one binding using the *Toggle Streaming* action and the control value toggle (with 127 on top and 75 on the bottom). This and other such adjustments would lower the number of responses to bindings, thus decreasing the performance issues.

Hope this helps!

**Comment by nhielost** [comment]
This discussion is being closed due to the implementation of the issue in question in version 3.0.0.
